lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -215,4 +216,11 @@ defmodule ExUnit.CLIFormatter do
       _ -> 80
     end
   end
+
+  defp print_logs(""), do: nil
+
+  defp print_logs(output) do
+    output = String.replace(output, "\n", "\n     ")
+    IO.puts(["     The following output was logged:\n" | output])
```

Comment:
With a custom formatter will their always be a leading newline? I think we may need to add indentation after the `\n` here.